docs/volto/bootstrap.md

```diff
@@ -37,7 +47,7 @@ Install `@plone/generator-volto`:
 npm install -g yo @plone/generator-volto
 ```
 
-## Bootstrapping A Project
+## Bootstrapping A project
```

Comment:
I'm concerned with this heading getting confused with the page title. Perhaps this?

```suggestion
## Create a Volto project
```